backend/main.py

```
@@ -193,6 +193,28 @@ def get_cors_origins():
     allow_headers=["*"],
 )
 
+# Global exception handler to ensure JSON error responses
+@app.exception_handler(Exception)
+async def global_exception_handler(request: Request, exc: Exception):
```

Ruff (0.14.3)

Unused function argument: `request`

(ARG001)